webos-js/package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "sample-webos-js",
+	"description": "Sample localization project",
+	"repository": "git@github.com:iLib-js/ilib-loctool-samples.git",
+	"license": "Apache-2.0",
+	"version": "1.0.0",
+	"scripts": {
+		"loc": "node node_modules/loctool/loctool.js -2 --xliffStyle custom",
```

Comment:
The node modules dir is already in the path inside of scripts, so you can just write:

```
loctool -2 --xliffStyle custom
```